src/archivist/index.js

```diff
@@ -23,7 +22,10 @@ export const AVAILABLE_EVENTS = [
   'versionRecorded',
   'firstVersionRecorded',
   'versionNotChanged',
-  'recordsPublished',
+  'startRefilterRun',
+  'refilterRunCompleted',
+  'startTrackingChangesRun',
+  'trackingChangesRunCompleted',
```

Comment:
Unify with other forms of events?
```suggestion
  'refilteringStarted',
  'refilteringCompleted',
  'trackingStarted',
  'trackingCompleted',
```